membership-attribute-service/app/monitoring/SentryLogging.scala

```diff
@@ -19,10 +18,12 @@ object SentryLogging extends SanitizedLogging {
       case Some(sentryDSN) =>
         logger.info(s"Initialising Sentry logging")
         Try {
-          val sentryClient = Sentry.init(sentryDSN)
+          Sentry.init(sentryDSN)
           val buildInfo: Map[String, String] = app.BuildInfo.toMap.view.mapValues(_.toString).toMap
           val tags = Map("stage" -> config.stage) ++ buildInfo
-          sentryClient.setTags(tags.asJava)
+          tags.foreach { case (key, value) =>
+            Sentry.setTag(key, value)
+          }
```

Comment:
this appeared to have changed in the later version of sentry lib, I just bumped it about 6 major versions (from 1.x.x. to 7.x.x).  I'm not sure we really use this functionality but I played around until it looked right in IntelliJ

Reply:
We do sent server side errors to sentry, but I am not sure we use it. I am not sure what we get that we don't get from `CloudWatch`- maybe it's worth adding to the noise reduction effort going on?

Reply:
I think sentry should remember all the exceptions only, group them together intelligently, and notify us somehow when something new goes wrong.  It sounds a lot better than cloudwatch's blunt metrics when used correctly.  But in reality I haven't used it, so it hasn't any value for me!